### PR TITLE
[MM-49559] - Fix emoji sprite in mozila firefox

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Fix emoji sprite display in Firefox

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49559

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: sass/components/_emojisprite.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
     height: 66px;
     background-repeat: no-repeat;
     cursor: pointer;
+    -moz-transform: scale(0.35);
     transform-origin: 0 0;
     // Using zoom for now as it results in less blurry emojis on Chrome - MM-34178
     zoom: 0.5;
```
Comment (koox00) [inline sass/components/_emojisprite.scss:13]: Should this be `-moz-transform: scale(0.5);` since `zoom: 0.5;`?

**Review by M-ZubairAhmed — APPROVED** [review]

File: sass/components/_emojisprite.scss (hunk 1 above)
Comment (nevyangelova) [reply sass/components/_emojisprite.scss:13]: this was the code before i deleted it, thinking it was obsolete

**Review by koox00 — APPROVED** [review]

**Comment by lieut-data** [comment]
@nevyangelova, will this also fix an issue with rendering stretched emojis in replies to posts?

image

**Comment by nevyangelova** [comment]
@lieut-data fixed :)

**Pull request merged**